pkg/containers/containers.go

```diff
@@ -183,12 +200,18 @@ func (c *Containers) populate() error {
 // NOTE: ALL given cgroup dir paths are stored in CgroupInfo map.
 func (c *Containers) CgroupUpdate(cgroupId uint64, path string, ctime time.Time) (CgroupInfo, error) {
 	containerId, runtime := getContainerIdFromCgroup(path)
+	metadata, err := c.enricher.Get(containerId, runtime, context.Background())
```

Comment:
if containerid is empty, there is no point in making this call

Reply:
You're right, i'll fix that.

Reply:
Done, let me know what you think